Library/Homebrew/cmd/gist-logs.rb

```diff
@@ -7,6 +7,9 @@
 #:     If `--new-issue` is passed, automatically create a new issue in the appropriate
 #:     GitHub repository as well as creating the Gist.
 #:
+#:     If `--test-logs` is passed, uploads the logs for the formula's last test run, instead
+#:     of the build logs.
```

Comment:
Could maybe always upload them all? Either way, not sure this makes sense to be an end-user option.